pkg/util/os/os_test.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package os
+
+import (
+	"testing"
+)
+
+func Test_environment_to_slice_with_nil_parameter_returns_empty_slice(t *testing.T) {
```

Comment:
these aren't really `os` tests, but are merely using a hardcoded `os.env` to iterate over. the setup can be more predictable by extracting this into a general util library.... and using a prepared input.